app/frontend/src/components/ApplicationList.vue

```diff
@@ -1,15 +1,22 @@
 <template>
   <v-container>
+    <div class="mx-auto mb-8 alert info pa-4" style="max-width: 800px;">
+      <h3 style="color: white !important">Attention GETOK users!</h3>
+      <p class="mt-4" style="color: white !important">Access to our hosted services is migrating to the API Services Portal at
+        <a style="color: white !important; text-decoration: underline !important; font-weight: bold" target=" _blank" href="https://api.gov.bc.ca/">api.gov.bc.ca</a>
+        <br>After December 2022, your credentials obtained through GETOK will no longer work.<br>For more guidance on this transition, please read 
+        <a style="color: white !important; text-decoration: underline !important; font-weight: bold" target=" _blank" href="https://github.com/bcgov/common-service-showcase/wiki/CSB-2022-005">more details on our Wiki.</a>
+      </p>
+    </div>
```

Comment:
As a future note, it's generally very bad practice to directly inject css style overrides into html. However, since getok is to be deprecated in the near future, we're letting it slide.